pkg/apis/app/v2/kieapp_types.go

```diff
@@ -356,6 +356,8 @@ type LDAPAuthConfig struct {
 	BindDN string `json:"bindDN,omitempty"`
 	// The JMX ObjectName of the JaasSecurityDomain used to decrypt the password.
 	JAASSecurityDomain string `json:"jaasSecurityDomain,omitempty"`
+	// A flag to set login module to optional. The default value is required
+	LoginModule string `json:"loginModule,omitempty"`
```

Comment:
are `required` and `optional` the only 2 possibilities here? if so, maybe we add them as enum options instead?
https://github.com/kiegroup/kie-cloud-operator/blob/master/pkg/apis/app/v2/kieapp_types.go#L19

Reply:
yes, only two options, this is actually a good idea. :+1: